fsspec/core.py

```diff
@@ -396,6 +397,9 @@ def url_to_fs(url, **kwargs):
     return fs, urlpath
 
 
+DEFAULT_EXPAND = conf.get("open_expand", True)
```

Comment:
I don't think `DEFAULT_EXPAND` should be True. It's a surprising behavior/footgun. It violates the Zen of Python `explicit is better than implicit`.